controllers/keyword_test.go

```diff
@@ -0,0 +1,101 @@
+package controllers_test
+
+import (
+	"fmt"
+	"net/http"
+
+	_ "google-scraper/initializers"
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("KeywordController", func() {
+	Describe("GET /keyword/:id", func() {
+		Context("given the user is an authenticated user", func() {
+			Context("given the keyword is belong to the user", func() {
+				It("returns 200 status code", func() {
+					user := FabricateUser("John", "john@example.com", "secret")
+					keyword := FabricateKeyword("Buy domain", false, &user)
+					url := fmt.Sprintf("/keyword/%d", keyword.Id)
+
+					response := MakeAuthenticatedRequest("GET", url, nil, &user)
+
+					Expect(response.StatusCode).To(Equal(http.StatusOK))
+				})
+			})
```

Comment:
Is it possible to have test case something like `returns correct keywords` and expect the body that returns from the request?

Reply:
As we discussed, we will ignore it.